components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/EmitOnStateChange.java

```diff
@@ -66,7 +66,7 @@
  * insert into AlertStream;
  */
 
-@Extension(name = "emitOnStateChange", namespace = "gateway/throttler", description = "The logger stream processor logs the "
+@Extension(name = "emitOnStateChange", namespace = "throttler", description = "The logger stream processor logs the "
```

Comment:
Lets use the old namespace

Reply:
due to an issue in new ballerina this namespace cannot be used